org.bndtools.templates.osgi/resources/templates/integration-test/bnd.bnd

```diff
@@ -1,9 +1,9 @@
 # This is the version of JUnit that will be used at build time and run time
-junit: org.apache.servicemix.bundles.junit;version="[4.11,5)"
+junit: org.apache.servicemix.bundles.junit;version=latest
 
 # This is the version of Mockito that will be used at build time and run time
-mockito: org.mockito.mockito-core;version="[1.9,2)",\
-  org.objenesis;version="[2.1,3)"
+mockito: org.mockito.mockito-core;version=latest,\
+  org.objenesis;version=latest
```

Comment:
move the test dependencies to a macro

Reply:
Waiting for clarification about  the macro above. Haven't defined a macro yet. It's probably simple, but an example would help.

Reply:
I have done this now.